standalone-metastore/pom.xml

```diff
@@ -123,6 +123,13 @@
         <artifactId>orc-core</artifactId>
         <version>${orc.version}</version>
       </dependency>
+      <dependency>
+        <groupId>com.fasterxml.jackson</groupId>
+        <artifactId>jackson-bom</artifactId>
+        <version>${jackson.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
       <dependency>
         <groupId>com.fasterxml.jackson.core</groupId>
         <artifactId>jackson-databind</artifactId>
```

Comment:
After adding the `dependencyManagement` section, the `jackson.version` for `jackson-databind` can be removed.

Reply:
Since this is already the dependencyManagement section. This jackson-databind version is no longer needed after the jackson-bom is added.